internify/src/App.js

```diff
@@ -56,6 +57,7 @@ function App() {
         <Route path="/profile">
           <AuthNavbar />
           <Profile />
+          <Footer />
```

Comment:
We want the Footer in the `Create` pages as well. We want the footer for all pages in fact.

Reply:
I adjusted it to make it less noticeable. There's a gap on both sides, and on the navbar as well, when one compares the edges of it to the purple background below. It may be something in the styling layout of the page itself. =(

Reply:
Your box shadow was the issue. You need to be using the box shadow from the Figma designs. I've demonstrated in the past on how you can get the box shadow.

Replace your box shadow style with this:
`  box-shadow: 0px 11px 20px 0px rgba(0, 0, 0, 0.25);`